docs/web-component.md

```diff
@@ -0,0 +1,888 @@
+<!-- meta
+{
+  "node": {
+    "label": "Web Component",
+    "category": {
+      "label": "Luigi Core"
+    },
+    "metaData": {
+      "categoryPosition": 2,
+      "position": 12
+    }
+  }
+}
+meta -->
+
+# Web Component
+
+<!-- add-attribute:class:success -->
+>**TIP:** You can find some examples how to write/include web components in our test application [Luigi Fiddle](https://fiddle.luigi-project.io).
+
+### Overview
+
+Luigi offers the possibility to open a micro frontend as a web component; if you want to have more information about web component, please have a look at the page: [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
+
+For no-complex micro frontend, it can be a good and fast alternative: all the frontend will be loaded in a single javascript file.
+
+In this page you wil find:
+-   [Navigation Configuration](#navigation-configuration) - how to configure web component in Luigi Core navigation
+-   [Write a Web Component](#write-a-web-component) - quick description how to write a Web Component compatible with Luigi Framework
+-   [Luigi Client for web component](#luigi-client-for-web-component) js object injected in that Web Component, to leverage some Luigi core features
+-   [Tip: how to inject HTML Template code in web component](#tip-how-to-inject-html-template-code-in-web-component) - recommendation how to inject the html in a Web Component
+-   [Tip: Loading a Web Component inside a Luigi WebComponent](#tip-loading-a-web-component-inside-a-luigi-webcomponent) - example how to include standard Web Components in your Luigi Web Component 
+-   [Tip: Example of Luigi Web Component using other standard Web Component and how to communicate with them](#tip-example-of-luigi-web-component-using-other-standard-web-component-and-how-to-communicate-with-them)
+    -   [File luigiExampleWC.js: the Luigi Web Compone](#file-luigiexamplewcjs-the-luigi-web-component) 
+    -   [File f_selectbox.js](#file-f_selectboxjs)
+    -   [File f_busy.js](#file-f_busyjs)
+    -   [File f_input.js](#file-f_inputjs)
+    -   [File wcEvent.js](#file-wceventjs)
+        
+            
+## Navigation Configuration
+
```

Comment:
I think this documentation part is missing the info regarding `settings` that are needed to enable wc, otherwise it would not work. It's already added in this example, it's just not documented here:
```
settings: {      
     ...
     navigation: {
        validWebcomponentUrls:['.*?']
     },
     experimental: {
          webcomponents:true
     }
}
```